install.ps1

```diff
@@ -960,83 +1038,69 @@ if (-not $noGui.IsPresent) {
     $installButton.width      = 97
     $installButton.height     = 37
     $installButton.DialogResult   = [System.Windows.Forms.DialogResult]::OK
-    $installButton.location   = New-Object System.Drawing.Point(750,800)
+    $installButton.location   = New-Object System.Drawing.Point(950,800)
     $installButton.Font       = New-Object System.Drawing.Font('Microsoft Sans Serif',12)
 
     $cancelButton            = New-Object system.Windows.Forms.Button
     $cancelButton.text       = "Cancel"
     $cancelButton.width      = 97
     $cancelButton.height     = 37
-    $cancelButton.location   = New-Object System.Drawing.Point(850,800)
+    $cancelButton.location   = New-Object System.Drawing.Point(1050,800)
     $cancelButton.Font       = New-Object System.Drawing.Font('Microsoft Sans Serif',12)
     $cancelButton.DialogResult = [System.Windows.Forms.DialogResult]::Cancel
+    
+    # Category selection panel
+    $categorySelectionLabel                = New-Object system.Windows.Forms.Label
+    $categorySelectionLabel.text           = "Select Categories to Include"
+    $categorySelectionLabel.AutoSize       = $true
+    $categorySelectionLabel.location       = New-Object System.Drawing.Point(30,60)
+    $categorySelectionLabel.Font           = New-Object System.Drawing.Font('Microsoft Sans Serif',10,[System.Drawing.FontStyle]([System.Drawing.FontStyle]::Bold))
+
+    $categoryPanel                = New-Object system.Windows.Forms.Panel
+    $categoryPanel.height         = 540
+    $categoryPanel.width          = 200
+    $categoryPanel.location       = New-Object System.Drawing.Point(30,80)
+    $categoryPanel.AutoScroll     = $true
+
+    # Create category checkboxes
+    $checkboxesCategories = New-Object System.Collections.Generic.List[System.Object]
+    $allCategories = $defaultExcludedCategories + @('Analysis','Browsers','Cloud','Debuggers','Decompilers','Document and File Format','Hex Editors','Java','Network','Utilities','Vulnerability Assessment')
```

Comment:
where are you taking these categories from? The list of categories is at https://github.com/mandiant/VM-Packages/blob/main/categories.txt